recirq/otoc/loschmidt/tilted_square_lattice/analysis-walkthrough.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<blockquote> Average (and compute the standard deviation) over random instances</blockquote><p>What does "random" mean here? My read of this sentence is that data is aggregated over a random subset of data samples, but <code>groupby_all_except()</code> seems to apply to all samples (or does it?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/ReCirq/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>this is a loschmidt echo thing where we do random instantiations of the circuit unitary U</p>

Reply:
<p>changed to "random circuit instances"</p>